test/WebSites/HtmlGenerationWebSite/Views/Shared/EditorTemplates/Item.cshtml

```diff
@@ -0,0 +1,4 @@
+﻿@using HtmlGenerationWebSite.Models
+@model Item
+@Html.Label(nameof(Model.Description))
+@Html.TextBoxFor(m => m.Description)
```

Comment:
To match the shared case more closely, add a `String.cshtml` template containing `@model string` plus the lines in `Common.cshtml`. The expectations shouldn't need to change...